source/views/transportation/other-modes/other-modes-detail.js

```diff
@@ -0,0 +1,71 @@
+// @flow
```

Comment:
Could you please rename these three files to remove the `other-modes-` prefix?

Reply:
Can we pick a convention for everything? I think this differs _per component_ these days.

Reply:
Yes. It does. One of the reasons for my reorganization. 

I don't see a need to duplicate the folder hierarchy in the file name. That's why I requested that you remove it.